apps/dev/pages/api/auth/[...nextauth].ts

```diff
@@ -150,6 +151,11 @@ export const authOptions: NextAuthOptions = {
       clientSecret: process.env.IDS4_SECRET,
       issuer: process.env.IDS4_ISSUER,
     }),
+    DuendeIDS6Provider({
+      clientId: process.env.DUENDE_IDS6_ID,
+      clientSecret: process.env.DUENDE_IDS6_SECRET,
+      issuer: process.env.DUENDE_IDS6_ISSUER,
+    }),
```

Comment:
```suggestion
    DuendeIDS6Provider({
      clientId: "interactive.confidential",
      clientSecret: "secret",
      issuer: "https://demo.duendesoftware.com",
    }),
```

Reply:
Would we not want this as environment variables? Interested in the choice to hardcode them this way instead!

Reply:
Alright cool yeah that's right given its only for demo purposes that seems fine to me :)